darwin/importer/importer.py

```diff
@@ -267,6 +274,159 @@ def _resolve_annotation_classes(
     return local_classes_not_in_dataset, local_classes_not_in_team
 
 
+def _import_properties(
+    metadata_path: Union[Path, bool],
+    client: "Client",
+    annotations: List[dt.Annotation],
+    annotation_class_ids_map: Dict[str, str],
+):
```

Comment:
can you docblock this, detail it mostly for internal as it's not really client facing code. But it would be good to have a semblance of the intendid flow, of what conditions cause it to raise and exit early. Ie when properties don't exist, mismatched etc.